From 7221bee66050a8fd1a806d6f154823b93e8a428c Mon Sep 17 00:00:00 2001
From: Sergio Prado <sergio.prado@e-labworks.com>
Date: Sat, 13 Jun 2020 10:03:41 +0000
Subject: [PATCH] Fix cross compilation issues

Signed-off-by: Sergio Prado <sergio.prado@e-labworks.com>
---
 sdl/configure.ac | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/sdl/configure.ac b/sdl/configure.ac
index 670ea05b279a..45f2655f525a 100644
--- a/sdl/configure.ac
+++ b/sdl/configure.ac
@@ -4,7 +4,7 @@ AC_REVISION([$Revision: 1.53 $])
 
 AC_CONFIG_SRCDIR([sdlmain.cpp])
 
-AC_CANONICAL_TARGET
+AC_CANONICAL_HOST
 
 AC_PROG_CC
 AC_PROG_CXX
@@ -38,7 +38,7 @@ AC_DEFUN([AC_S9X_COMPILER_FLAG],
 				return (argc);
 			}
 		],
-		[snes9x_cv_option_$2="yes"], [snes9x_cv_option_$2="no"])
+		[snes9x_cv_option_$2="yes"], [snes9x_cv_option_$2="no"], [snes9x_cv_option_$2="yes"])
 	])
 
 	CXXFLAGS="[$]OLD_CXXFLAGS"
@@ -240,8 +240,8 @@ AC_CACHE_VAL([snes9x_cv_sdl],
 ])
 
 if test "x$snes9x_cv_sdl" = "xyes"; then
-  	S9XFLGS="$S9XFLGS `sdl-config --cflags`"
-	S9XLIBS="$S9XLIBS `sdl-config --libs`"
+	S9XFLGS="$S9XFLGS `pkg-config --cflags sdl`"
+	S9XLIBS="$S9XLIBS `pkg-config --libs sdl`"
 else
 	AC_MSG_ERROR([SDL is required.])
 fi
@@ -332,7 +332,7 @@ AC_DEFUN([AC_S9X_CHECK_SAR],
 			return (i < 0 ? 0 : 1);
 		}
 	],
-	[snes9x_sar_$1="yes"], [snes9x_sar_$1="no"])
+	[snes9x_sar_$1="yes"], [snes9x_sar_$1="no"], [snes9x_sar_$1=$GCC])
 
 	CXXFLAGS="[$]OLD_CXXFLAGS"
 
-- 
2.17.1

