--- advancemame-1.2/advance/advance.mak-org	2015-06-13 14:59:22.414622362 +0000
+++ advancemame-1.2/advance/advance.mak	2015-06-13 15:04:00.642622329 +0000
@@ -290,83 +290,83 @@
 pkgdocdir = $(docdir)/advance
 
 install-dirs:
-	-$(INSTALL_PROGRAM_DIR) $(bindir)
-	-$(INSTALL_DATA_DIR) $(pkgdir)
-	-$(INSTALL_DATA_DIR) $(pkgdocdir)
-	-$(INSTALL_MAN_DIR) $(mandir)/man1
+	-$(INSTALL_PROGRAM_DIR) $(DESTDIR)$(bindir)
+	-$(INSTALL_DATA_DIR) $(DESTDIR)$(pkgdir)
+	-$(INSTALL_DATA_DIR) $(DESTDIR)$(pkgdocdir)
+	-$(INSTALL_MAN_DIR) $(DESTDIR)$(mandir)/man1
 ifneq ($(wildcard $(EMUSRC)),)
-	-$(INSTALL_DATA_DIR) $(pkgdir)/rom
-	-$(INSTALL_DATA_DIR) $(pkgdir)/sample
-	-$(INSTALL_DATA_DIR) $(pkgdir)/artwork
-	-$(INSTALL_DATA_DIR) $(pkgdir)/image
-	-$(INSTALL_DATA_DIR) $(pkgdir)/crc
+	-$(INSTALL_DATA_DIR) $(DESTDIR)$(pkgdir)/rom
+	-$(INSTALL_DATA_DIR) $(DESTDIR)$(pkgdir)/sample
+	-$(INSTALL_DATA_DIR) $(DESTDIR)$(pkgdir)/artwork
+	-$(INSTALL_DATA_DIR) $(DESTDIR)$(pkgdir)/image
+	-$(INSTALL_DATA_DIR) $(DESTDIR)$(pkgdir)/crc
 endif
 
 install-data: $(INSTALL_DATAFILES)
 ifdef INSTALL_DATAFILES
 	@for i in $(INSTALL_DATAFILES); do \
-		echo "$(INSTALL_DATA) $$i $(pkgdir)"; \
-		$(INSTALL_DATA) $$i $(pkgdir); \
+		echo "$(INSTALL_DATA) $$i $(DESTDIR)$(pkgdir)"; \
+		$(INSTALL_DATA) $$i $(DESTDIR)$(pkgdir); \
 	done
 endif
 
 uninstall-data:
 ifdef INSTALL_DATAFILES
 	@for i in $(notdir $(INSTALL_DATAFILES)); do \
-		rm -f $(pkgdir)/$$i; \
+		rm -f $(DESTDIR)$(pkgdir)/$$i; \
 	done
 endif
 
 install-bin: $(INSTALL_BINFILES)
 	@for i in $(INSTALL_BINFILES); do \
-		echo "$(INSTALL_PROGRAM) $$i $(bindir)"; \
-		$(INSTALL_PROGRAM) $$i $(bindir); \
+		echo "$(INSTALL_PROGRAM) $$i $(DESTDIR)$(bindir)"; \
+		$(INSTALL_PROGRAM) $$i $(DESTDIR)$(bindir); \
 	done
 
 uninstall-bin:
 	@for i in $(notdir $(INSTALL_BINFILES)); do \
-		rm -f $(bindir)/$$i; \
+		rm -f $(DESTDIR)$(bindir)/$$i; \
 	done
 
 install-doc: $(INSTALL_DOCFILES)
 ifdef INSTALL_DOCFILES
 	@for i in $(INSTALL_DOCFILES); do \
-		echo "$(INSTALL_DATA) $$i $(pkgdocdir)"; \
-		$(INSTALL_DATA) $$i $(pkgdocdir); \
+		echo "$(INSTALL_DATA) $$i $(DESTDIR)$(pkgdocdir)"; \
+		$(INSTALL_DATA) $$i $(DESTDIR)$(pkgdocdir); \
 	done
 endif
 
 uninstall-doc:
 ifdef INSTALL_DOCFILES
 	@for i in $(notdir $(INSTALL_DOCFILES)); do \
-		rm -f $(pkgdocdir)/$$i; \
+		rm -f $(DESTDIR)$(pkgdocdir)/$$i; \
 	done
 endif
 
 install-man: $(INSTALL_MANFILES)
 ifdef INSTALL_MANFILES
 	@for i in $(INSTALL_MANFILES); do \
-		echo "$(INSTALL_DATA) $$i $(mandir)/man1"; \
-		$(INSTALL_DATA) $$i $(mandir)/man1; \
+		echo "$(INSTALL_DATA) $$i $(DESTDIR)$(mandir)/man1"; \
+		$(INSTALL_DATA) $$i $(DESTDIR)$(mandir)/man1; \
 	done
 endif
 
 uninstall-man:
 ifdef INSTALL_MANFILES
 	@for i in $(notdir $(INSTALL_MANFILES)); do \
-		rm -f $(mandir)/man1/$$i; \
+		rm -f $(DESTDIR)$(mandir)/man1/$$i; \
 	done
 endif
 
 uninstall-dirs:
 ifneq ($(wildcard $(EMUSRC)),)
-	-rmdir $(pkgdir)/rom
-	-rmdir $(pkgdir)/sample
-	-rmdir $(pkgdir)/artwork
-	-rmdir $(pkgdir)/image
-	-rmdir $(pkgdir)/crc
-	-rmdir $(pkgdir)
-	-rmdir $(pkgdocdir)
+	-rmdir $(DESTDIR)$(pkgdir)/rom
+	-rmdir $(DESTDIR)$(pkgdir)/sample
+	-rmdir $(DESTDIR)$(pkgdir)/artwork
+	-rmdir $(DESTDIR)$(pkgdir)/image
+	-rmdir $(DESTDIR)$(pkgdir)/crc
+	-rmdir $(DESTDIR)$(pkgdir)
+	-rmdir $(DESTDIR)$(pkgdocdir)
 endif
 
 install: install-dirs install-bin install-data install-doc install-man
