--- advancemame-1.2/configure.ac~org	2015-06-10 16:12:31.451792611 +0000
+++ advancemame-1.2/configure.ac	2015-06-10 16:12:44.955792609 +0000
@@ -146,19 +146,7 @@
 		CFLAGS="-O0 -g -Wall -Wno-sign-compare -Wno-unused"
 	elif test $ac_enable_test = yes ; then
 		CFLAGS="-O0 -Wall -Wno-sign-compare -Wno-unused"
-	else
-		CFLAGS=""
-		AC_CHECK_CC_OPT([-march=native], [CFLAGS="$CFLAGS -march=native"], [])
-		AC_CHECK_CC_OPT([-O2], [CFLAGS="$CFLAGS -O2"], [CFLAGS="$CFLAGS -O"])
-		AC_CHECK_CC_OPT([-fomit-frame-pointer], [CFLAGS="$CFLAGS -fomit-frame-pointer"], [])
-		# Code is not ready for strict-aliasing. It seems working, but who know...
-		AC_CHECK_CC_OPT([-fno-strict-aliasing], [CFLAGS="$CFLAGS -fno-strict-aliasing"], [])
-		# We are not interested in stack protection
-		AC_CHECK_CC_OPT([-fno-stack-protector], [CFLAGS="$CFLAGS -fno-stack-protector"], [])
-		AC_CHECK_CC_OPT([-Wall], [CFLAGS="$CFLAGS -Wall"], [])
-		AC_CHECK_CC_OPT([-Wno-sign-compare], [CFLAGS="$CFLAGS -Wno-sign-compare"], [])
-		AC_CHECK_CC_OPT([-Wno-unused], [CFLAGS="$CFLAGS -Wno-unused"], [])
	fi
 	if test ! -z $ac_host_cflags; then
 		CFLAGS="$ac_host_cflags $CFLAGS"
 	fi
