From 4b54416ae67ecfc67d7c5428f7ce65c1256add62 Mon Sep 17 00:00:00 2001
From: Sergio Prado <sergio.prado@e-labworks.com>
Date: Fri, 12 Jun 2020 00:27:27 +0000
Subject: [PATCH] Fix cross compilation issues

Signed-off-by: Sergio Prado <sergio.prado@e-labworks.com>
---
 unix/configure.ac | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/unix/configure.ac b/unix/configure.ac
index 382a59de390f..1a2f8bc3d8ff 100644
--- a/unix/configure.ac
+++ b/unix/configure.ac
@@ -4,7 +4,7 @@ AC_REVISION([$Revision: 1.60 $])
 
 AC_CONFIG_SRCDIR([unix.cpp])
 
-AC_CANONICAL_TARGET
+AC_CANONICAL_HOST
 
 AC_PROG_CC
 AC_PROG_CXX
@@ -38,7 +38,7 @@ AC_DEFUN([AC_S9X_COMPILER_FLAG],
 				return (argc);
 			}
 		],
-		[snes9x_cv_option_$2="yes"], [snes9x_cv_option_$2="no"])
+		[snes9x_cv_option_$2="yes"], [snes9x_cv_option_$2="no"], [snes9x_cv_option_$2="yes"])
 	])
 
 	CXXFLAGS="[$]OLD_CXXFLAGS"
@@ -100,7 +100,7 @@ AC_MSG_CHECKING([whether the OS is Linux])
 
 AC_CACHE_VAL([snes9x_cv_linux_os],
 [
-	case "$target" in
+	case "$host" in
 		*-*-linux*)
 			snes9x_cv_linux_os="yes"
 			;;
@@ -365,7 +365,7 @@ AC_DEFUN([AC_S9X_CHECK_SAR],
 			return (i < 0 ? 0 : 1);
 		}
 	],
-	[snes9x_sar_$1="yes"], [snes9x_sar_$1="no"])
+	[snes9x_sar_$1="yes"], [snes9x_sar_$1="no"], [snes9x_sar_$1=$GCC])
 
 	CXXFLAGS="[$]OLD_CXXFLAGS"
 
-- 
2.17.1

