diff --git a/configure.ac b/configure.ac
index bc07e2fd2dfd..045bbb1493dd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -395,13 +395,13 @@ CheckJACK()
 AC_HELP_STRING([--enable-jack], [support the JACK audio API [[default=yes]]]),
                   , enable_jack=yes)
     if test x$enable_jack = xyes; then
-        PKG_CHECK_MODULES(JACK, jack, [have_jack=yes], [have_jack=no])
+        have_jack=yes
 
         if test x$have_jack = xyes; then
                 AM_CONDITIONAL(HAVE_JACK, true)
                 AC_DEFINE([HAVE_JACK], [1], [Define if we are compiling with JACK support.])
-                LIBS="$LIBS $JACK_LIBS"
-                CPPFLAGS="$CPPFLAGS $JACK_CFLAGS"
+                LIBS="$LIBS `pkg-config --libs jack`"
+                CPPFLAGS="$CPPFLAGS `pkg-config --cflags jack`"
         fi
     fi
 }
@@ -476,7 +476,8 @@ else
 	AC_DEFINE([PSS_STYLE],[1], [Defines the filesystem path-separator type.])
 
 	if test x$with_libsndfile = xyes; then
-		PKG_CHECK_MODULES(SNDFILE, sndfile >= 1.0.2, [], AC_MSG_ERROR([*** libsndfile >= 1.0.2 not found!]))
+        SNDFILE_LIBS="-lsndfile"
+        SNDFILE_CFLAGS=""
                 AC_DEFINE([HAVE_LIBSNDFILE], [1], [Define if we are compiling with libsndfile support.])
 		AM_CONDITIONAL(HAVE_LIBSNDFILE, true)
 	fi
