diff --git a/src/tests.cpp b/src/tests.cpp
index 1cf0ee0b4f5d..9084f15271a7 100644
--- a/src/tests.cpp
+++ b/src/tests.cpp
@@ -1050,7 +1050,7 @@ static void zlib_test(void)
  if((2 << ((cfl >> 6) & 0x3)) != sizeof(z_off_t))
  {
   assert(sizeof(z_off64_t) == 8);
-  assert(&gztell == &gztell64);
+  assert(&gztell == (off_t (*)(gzFile))&gztell64);
  }
  #else
  assert((2 << ((cfl >> 6) & 0x3)) == sizeof(z_off_t));
