From 73de49cf128921344e1c95a49f272047fd499747 Mon Sep 17 00:00:00 2001
From: Jools Wills <jools@oxfordinspire.co.uk>
Date: Thu, 12 Feb 2015 17:51:13 +0000
Subject: [PATCH 3/5] added missing \n

---
 src/video/dispmanx/SDL_dispmanxvideo.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/video/dispmanx/SDL_dispmanxvideo.c b/src/video/dispmanx/SDL_dispmanxvideo.c
index 5daf76325bf3..e4a2c580730b 100644
--- a/src/video/dispmanx/SDL_dispmanxvideo.c
+++ b/src/video/dispmanx/SDL_dispmanxvideo.c
@@ -245,7 +245,7 @@ static SDL_Surface *DISPMANX_SetVideoMode(_THIS, SDL_Surface *current, int width
 		int dst_xpos  = (dispvars->amode.width - dst_width) / 2;
 		int dst_ypos  = (dispvars->amode.height - dst_height) / 2;
 
-		printf ("Dispmanx: Scaling to %d x %d", dst_width, dst_height);
+		printf ("Dispmanx: Scaling to %d x %d\n", dst_width, dst_height);
 
 		vc_dispmanx_rect_set( &(dispvars->dst_rect), dst_xpos, dst_ypos,
 		dst_width , dst_height );
-- 
1.9.1

