From 274c1af48ed93c29aeb878ec9208562cae167f5b Mon Sep 17 00:00:00 2001
From: Sergio Prado <sergio.prado@e-labworks.com>
Date: Sat, 13 Jun 2020 09:23:25 +0000
Subject: [PATCH] Fix cross compilation issues

Signed-off-by: Sergio Prado <sergio.prado@e-labworks.com>
---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 9235d9740ae2..f0c21544785b 100644
--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,9 @@
 
-CXX = g++
+CXX ?= g++
 
 INCLDIRS=-Iunix -Iunix/input -Iunzip -I.
-LIBDIRS=-L/usr/lib/arm-linux-gnueabihf
 
-OPTIMISE= -D_ZAURUS -O3 -march=armv6zk -mcpu=arm1176jzf-s -mtune=arm1176jzf-s -mfpu=vfp -mfloat-abi=hard -ffast-math -fstrict-aliasing -fomit-frame-pointer
+OPTIMISE= -D_ZAURUS -O3 -mfpu=vfp -mfloat-abi=hard -ffast-math -fstrict-aliasing -fomit-frame-pointer
 
 UNZIPDEFINES=-DUNZIP_SUPPORT
 SOUNDDEFINES=-DSPC700_C
@@ -14,6 +13,7 @@ CXXFLAGS = $(OPTIMISE) \
 -DZLIB \
 -DVAR_CYCLES \
 -DCPU_SHUTDOWN \
+-Wno-narrowing \
 -DSPC700_SHUTDOWN \
 $(SOUNDDEFINES) \
 $(UNZIPDEFINES) \
-- 
2.17.1

