diff --git a/Makefile b/Makefile
index 67b1a10a7a81..620072cc1b9c 100644
--- a/Makefile
+++ b/Makefile
@@ -172,7 +172,7 @@ config.mak: $(srcdir)/configure
 
 install: all
 	$(INSTALL) -d "$(DESTDIR)$(BINDIR)"
-	$(INSTALL) -c -s -m 755 "$(srcdir)/stella$(EXEEXT)" "$(DESTDIR)$(BINDIR)/stella$(EXEEXT)"
+	$(INSTALL) -c -m 755 "$(srcdir)/stella$(EXEEXT)" "$(DESTDIR)$(BINDIR)/stella$(EXEEXT)"
 	$(INSTALL) -d "$(DESTDIR)$(DOCDIR)"
 	$(INSTALL) -c -m 644 "$(srcdir)/Announce.txt" "$(srcdir)/Changes.txt" "$(srcdir)/Copyright.txt" "$(srcdir)/License.txt" "$(srcdir)/README-SDL.txt" "$(srcdir)/Readme.txt" "$(srcdir)/Todo.txt" "$(srcdir)/docs/index.html" "$(srcdir)/docs/debugger.html" "$(DESTDIR)$(DOCDIR)/"
 	$(INSTALL) -d "$(DESTDIR)$(DOCDIR)/graphics"
diff --git a/configure b/configure
index 419eb8a5fe10..8f453b9f5d62 100755
--- a/configure
+++ b/configure
@@ -334,6 +334,7 @@ mingw32-cross)
 	_host_cpu=`echo $guessed_host | sed 's/^\([^-]*\)-\([^-]*\)-\(.*\)$/\1/'`
 	_host_os=`echo $guessed_host | sed 's/^\([^-]*\)-\([^-]*\)-\(.*\)$/\3/'`
 	_host_vendor=`echo $guessed_host | sed 's/^\([^-]*\)-\([^-]*\)-\(.*\)$/\2/'`
+    _host_prefix=$_host
 	;;
 esac
 
@@ -375,20 +376,27 @@ esac
 # Determine the C++ compiler
 #
 echo_n "Looking for C++ compiler... "
-if test -n "$_host"; then
-	compilers="$CXX $_host_prefix-g++ $_host_prefix-c++ $_host_cpu-$_host_os-g++ $_host_cpu-$_host_os-c++"
-else
-	compilers="$CXX g++ c++"
-fi
 
-for compiler in $compilers; do
-	if test_compiler $compiler; then
-		CXX=$compiler
-		echo $CXX
-		break
-	fi
-done
-if test -z $CXX; then
+if test_compiler "$CXX"; then
+    echo "$CXX"
+else
+    if test -n "$_host"; then
+        compilers="$_host_prefix-g++ $_host_prefix-c++ $_host_cpu-$_host_os-g++ $_host_cpu-$_host_os-c++"
+    else
+        compilers="g++ c++"
+    fi
+
+    for compiler in $compilers; do
+        echo "testing compile $compiler"
+        if test_compiler $compiler; then
+		    CXX=$compiler
+        	echo $CXX
+		    break
+        fi
+    done
+fi
+
+if test -z "$CXX"; then
 	echo "none found!"
 	exit 1
 fi
@@ -529,8 +537,9 @@ if test -n "$_host"; then
 			_host_os=win32
 			;;
 		*)
-			echo "Cross-compiling to unknown target, please add your target to configure."
-			exit 1
+            echo "Cross-compiling for Yocto Project based systems."
+            DEFINES="$DEFINES -DUNIX"
+            _host_os=unix
 			;;
 	esac
 	
@@ -734,7 +743,7 @@ fi
 # Now, add the appropriate defines/libraries/headers
 #
 echo
-find_sdlconfig
+#find_sdlconfig
 
 SRC="src"
 CORE="$SRC/emucore"
@@ -750,7 +759,7 @@ ZLIB="$SRC/zlib"
 
 INCLUDES="-I$CORE -I$COMMON -I$TV -I$GUI"
 
-INCLUDES="$INCLUDES `$_sdlconfig --cflags`"
+INCLUDES="$INCLUDES `pkg-config --cflags sdl`"
 if test "$_build_static" = yes ; then
 	_sdl_conf_libs="--static-libs"
 	LDFLAGS="-static $LDFLAGS"
@@ -758,7 +767,7 @@ else
 	_sdl_conf_libs="--libs"
 fi
 
-LIBS="$LIBS `$_sdlconfig $_sdl_conf_libs`"
+LIBS="$LIBS `pkg-config --libs sdl`"
 LD=$CXX 
 case $_host_os in
 		unix)
